modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidator.java

```diff
@@ -190,14 +190,14 @@ private void validateParameterGroup(ParameterModel parameterModel)
         if (parameterGroup != null)
         {
             parameterGroup.getGroups().stream()
-                    .map(p -> p.getField())
-                    .forEach(field -> {
-                        if (!isInstantiable(field.getType(), false))
+                    .forEach(parameter -> {
+                        if (!isInstantiable(parameter.getType()))
                         {
                             throw new IllegalParameterModelDefinitionException(
-                                    format("Parameter group type should be instantiable. Affected field [%s] in [%s].",
-                                           field.getName(),
-                                           field.getAnnotatedType()));
+                                    format("ParameterGroup type should be non abstract with a default constructor, however type '%s' was received. Affected field [%s] in [%s].",
+                                           parameter.getType().getName(),
+                                           parameter.getField().getName(),
```

Comment:
why do you insist with the field?? the field is an internal thing which has nothing to do with this. If the `@ParameterGroup` is on an operation, what does the field tell the user. Just use parameter.getType() and nothing else